### ISSUE TITLE
[Linux] Cannot input controls to Ryu!

### ISSUE THREAD

**Issue by amunabv** [Ryujinx/Ryujinx#1180]
Hey,
On my Ryu install I can't actually input controls to games. Not even with keyboard!

I tried mainline and acnh4 build; Ryu interface works fine, (although not all settings seem to be saved yet ex. Game Directory locations). I can even remap keyboard buttons.

But when the game is started I can't do anything!!!

In the discord other arch users said they didn't have issues so I really don't know what's wrong :( Help!

[Ryujinx_2020-04-30_13-47-10.log](https://github.com/Ryujinx/Ryujinx/files/4558040/Ryujinx_2020-04-30_13-47-10.log)

I see this (animated, so no freeze):
![afbeelding](https://user-images.githubusercontent.com/9385838/80704739-667f8b00-8ae5-11ea-8ea7-7f12f2cd3f6a.png)

Arch 5.6.8, i5 6600k, 16GB, GTX980Ti

**Comment by Thog** [comment]
Can you provide your DE, WM and package list?

**Comment by rkunschke** [comment]
Same Problem here on Arch with plasma 5 on xorg. I tried to use a xinput controller and it didn't work.
Keyboardinbput works for me.

**Comment by AcK77** [comment]
Have you found how to fix that? It’s due to a bad setting? Or the issue is still here?

**Comment by amunabv** [comment]
> Have you found how to fix that? It’s due to a bad setting? Or the issue is still here?

It was running in the end but it was unclear to me what actually solved the issue. I remember at some point I was able to map the DualShock4 controller 1 button at a time, restart ryu and map another one etc. (because after configuring a button, it would get 'stuck' to that button and not change like from Button 1 to Button 2. So i had to save, restart Ryu and then map the next one). After I did all that stuff it was working for me. Maybe there are some permission settings at play here. I had pretty much just setup Arch about for 2 weeks. (ex-Windows user)
